lib/OIDCAuth.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+class SparkAPI_OIDCAuth extends SparkAPI_Core implements SparkAPI_AuthInterface
+{
+    const SPARK_PROVIDER = 'https://sparkplatform.com';
+    public $oauth_key;
+    public $oauth_secret;
+    public $redirect_uri;
+    public $auth_mode;
+    public $access_token;
+    public $last_token;
+    public $refresh_token;
+
+    public function __construct($oauth_key, $oauth_secret, $redirect_uri)
+    {
+        $this->auth_mode = 'OIDC';
```

Comment:
Nope, but all the other Auth classes do it, so I figured we may as well stay consistent in case it's used at some point: https://github.com/sparkapi/sparkapi4p2/blob/master/lib/APIAuth.php#L12